README.rst

```diff
@@ -1,6 +1,6 @@
 .. image:: http://docs.celeryproject.org/en/latest/_images/celery-banner-small.png
 
-|build-status| |coverage| |license| |wheel| |pyversion| |pyimp| |ocbackerbadge| |ocsponsorbadge|
+|build-status| |coverage| |license| |wheel| |pyversion| |pyimp| |ocbackerbadge| |ocsponsorbadge| |[![Downloads](https://pepy.tech/badge/celery)](https://pepy.tech/project/celery)| | [![Downloads](https://pepy.tech/badge/celery/month)](https://pepy.tech/project/celery/month) |
```

Comment:
The images do not seem to be rendered correctly. I think you need reStructured Text [syntax](http://docutils.sourceforge.net/docs/ref/rst/directives.html#image).